packages/auth/src/platform_browser/index.ts

```diff
@@ -41,14 +77,30 @@ export function getAuth(app: FirebaseApp = getApp()): Auth {
     return provider.getImmediate();
   }
 
-  return initializeAuth(app, {
+  const auth = initializeAuth(app, {
     popupRedirectResolver: browserPopupRedirectResolver,
     persistence: [
       indexedDBLocalPersistence,
       browserLocalPersistence,
       browserSessionPersistence
     ]
   });
+
+  const authTokenSyncUrl = getExperimentalSetting('authTokenSyncURL');
+  if (authTokenSyncUrl) {
+    const mintCookie = mintCookieFactory(authTokenSyncUrl);
+    beforeAuthStateChanged(auth, mintCookie, () => {
+      void mintCookie(auth.currentUser);
```

Comment:
Why is the Promise ignored? Please add a comment.

Reply:
Looks like I fixed an eslint error the wrong way. `beforeAuthStateChanged` does seem to await its callbacks so returned the promise instead.